service/ibmmq-plugin/pom.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.solace.maas</groupId>
+        <artifactId>maas-event-management-agent-parent</artifactId>
+        <version>1.3.2-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>ibmmq-plugin</artifactId>
+    <version>0.0.2-SNAPSHOT</version>
```

Comment:
@gregmeldrum question for you, shouldn't all plugin versions match `maas-event-management-agent-parent` based on current architecture of EMA?